module/merintr/inventry.c

```diff
@@ -1062,6 +1076,39 @@ Bool InventoryDropCurrentItem(room_contents_node *container)
    return True;
 }
 /************************************************************************/
+/*
+ * InventoryMoveCurrentItem:  Move the item with the inventory cursor, if any.
```

Comment:
Improve documentation: move it to where?  Move it to the item at the given (x, y) coordinates, if any.